packages/cms-lib/lib/config.js

```diff
@@ -271,11 +271,12 @@ const updatePortalConfig = configOptions => {
       tokenInfo,
     };
   }
+
   const env = getConfigEnv(environment || (portalConfig && portalConfig.env));
   const mode = defaultMode && defaultMode.toLowerCase();
   const nextPortalConfig = {
     ...portalConfig,
-    name,
+    name: name || (portalConfig && portalConfig.name),
```

Comment:
We should probably have some unit tests for `updatePortalConfig`.

Reply:
Added tests for portal naming/renaming as well as various others for updatePortalConfig.